README.md

```diff
@@ -1,2 +1,74 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# OAM Kubernetes Runtime
+
+The OAM Kubernetes runtime project is a collection of Golang helper libraries and OAM api type
+definitions. 
+
+It is designed to help OAM platform builders rather than being used directly by developers
+or end-users. We would recommend end-users to check
+[Crossplane  official  documentation](https://crossplane.github.io/docs) instead.
+
+In addition, we created this library with the following goals in mind  
+* All OAM Kubernetes platform builders use and contribute to this library. 
+* The wide adoption of this library can facilitate workload/trait interoperability among OAM
+ Kubernetes platform builders.
+ 
+## Prerequisites
+
+1. Golang version 1.12+
+2. Kubernetes version v1.15+ with kubectl configured
+
+## Getting started
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and Clone this project
+
+* Build the library 
+
+```shell
+make submodules 
+
+make
+```
```

Comment:
`make test` and `make reviewable` are also handy targets that may be worth documenting. The former runs the unit tests, while the latter runs linters, generation, etc to catch some of the things that might cause CI to fail.

Reply:
sure, we can add some instructions in the "how to contribute" section.